data-and-functions-2/src/functions/getTotalPriceForOrder.js

```diff
@@ -0,0 +1,18 @@
+import getProductById from './getProductById'
```

Comment:
Notice how similar this function (`getTotalPriceForOrder`) is to `getProductsForOrder`. Can you import that function and use it to get the total price?